from .._core_exceptions import LograderError

class LograderIOError(LograderError):
    pass